app/views/plantings/_thumbnail.html.haml

```diff
@@ -4,7 +4,9 @@
   .panel-body
     .row
       .col-xs-12.col-md-4
-        = link_to image_tag((planting.default_photo ?  planting.default_photo.thumbnail_url : 'placeholder_150.png'), :alt => planting.crop_id, :class => 'img'), planting
+        = link_to image_tag((planting.default_photo ? planting.default_photo.thumbnail_url : 'placeholder_150.png'),
```

Comment:
This image-tag code is very similar to the image-tag code in `thumbnail_with_popover`. Possibly we should unify them at some point, but I think it's out of scope for this PR.